setup.cfg

```diff
@@ -6,6 +6,7 @@ skip =
 include_trailing_comma = true
 multi_line_output = 5
 wrap_length = 120
+known_third_party = mypy_django_plugin
```

Comment:
This is a `FIRSTPARTY` code I would say. Because it is written right here: https://github.com/typeddjango/django-stubs/tree/master/mypy_django_plugin

Reply:
Fixed.